Exiled.API/Extensions/MirrorExtensions.cs

```diff
@@ -0,0 +1,299 @@
+        /// </summary>
+        /// <param name="player">Only this player can see info.</param>
+        /// <param name="target">Target to set info.</param>
+        /// <param name="info">Setting info.</param>
+        public static void SetPlayerInfoForTargetOnly(this Player player, Player target, string info) => SendFakeSyncVar(player, target.ReferenceHub.networkIdentity, typeof(NicknameSync), nameof(NicknameSync.Network_customPlayerInfoString), info);
+
+        /// <summary>
+        /// Change <see cref="Player"/> character model for appearance.
+        /// It will continue until <see cref="Player"/>'s <see cref="RoleType"/> changes.
+        /// </summary>
+        /// <param name="player">Player to change.</param>
+        /// <param name="type">Model type.</param>
+        public static void ChangeAppearance(this Player player, RoleType type)
+        {
+            foreach (var target in Player.List.Where(x => x != player))
+                SendFakeSyncVar(target, player.ReferenceHub.networkIdentity, typeof(CharacterClassManager), nameof(CharacterClassManager.NetworkCurClass), (sbyte)type);
+        }
+
+        /// <summary>
+        /// Send CASSIE announcement that only <see cref="Player"/> can hear.
+        /// It will continue until <see cref="Player"/>'s <see cref="RoleType"/> changes.
+        /// </summary>
+        /// <param name="player">Target to send.</param>
+        /// <param name="words">Announcement words.</param>
+        /// <param name="makeHold">Same on <see cref="Cassie.Message(string, bool, bool)"/>'s isHeld.</param>
+        /// <param name="makeNoise">Same on <see cref="Cassie.Message(string, bool, bool)"/>'s isNoisy.</param>
+        public static void PlayCassieAnnouncement(this Player player, string words, bool makeHold = false, bool makeNoise = true) => SendFakeTargetRpc(player, RespawnEffectsController.AllControllers.First().netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), words, makeHold, makeNoise);
+
+        /// <summary>
+        /// Send fake values to client's <see cref="Mirror.SyncVarAttribute"/>.
+        /// </summary>
+        /// <param name="target">Target to send.</param>
+        /// <param name="behaviorOwner"><see cref="Mirror.NetworkIdentity"/> of object that owns <see cref="Mirror.NetworkBehaviour"/>.</param>
+        /// <param name="targetType"><see cref="Mirror.NetworkBehaviour"/>'s type.</param>
+        /// <param name="propertyName">Property name starting with Network.</param>
+        /// <param name="value">Value of send to target.</param>
+        public static void SendFakeSyncVar(Player target, NetworkIdentity behaviorOwner, Type targetType, string propertyName, object value)
+        {
+            Action<NetworkWriter> customSyncVarGenerator = (targetWriter) =>
+            {
+                targetWriter.WritePackedUInt64(SyncVarDirtyBits[$"{targetType.Name}.{propertyName}"]);
```

Comment:
Shouldn't we use `SyncVarDirtyBits.TryGetValue` to avoid `KeyNotFoundException`s?